src/EnergyPlus/Furnaces.cc

```diff
@@ -6929,7 +6895,6 @@ namespace Furnaces {
 
                     // Set the input parameters for CalcFurnaceOutput
                     thisFurnace.CompPartLoadRatio = 0.0; // compressor off
-                    thisFurnace.WSHPRuntimeFrac = 0.0;
```

Comment:
Runtime fractions are now calculated in the coil code based on their PLF curves.